README.md

```diff
@@ -1,46 +1,72 @@
-# Event Publisher Proxy
+# Eventing Publisher Proxy
 
 ## Overview
 
-The Event Publisher Proxy receives legacy and Cloud Event publishing requests from the cluster workloads (microservice or Serverless functions) and redirects them to the Enterprise Messaging Service Cloud Event Gateway. It also fetches a list of subscriptions for a connected application.
+The Eventing Publisher Proxy receives legacy and Cloud Event publishing requests from the cluster workloads (microservice or Serverless functions) and redirects them to the Enterprise Messaging Service Cloud Event Gateway. It also fetches a list of subscriptions for a connected application.
 
 ## Prerequisites
 
-- Go modules
-- [ko](https://github.com/google/ko)
-- lint
-  - Install lint on the local environment:
-```bash
-curl -sfL https://install.goreleaser.com/github.com/golangci/golangci-lint.sh | bash -s -- -b $GOPATH/bin
-```    
+- [Go](https://go.dev/)
+- [Docker](https://www.docker.com/)
+- [kubectl](https://kubernetes.io/docs/tasks/tools/)
+- [kustomize](https://kustomize.io/)
+- Access to Kubernetes cluster ([k3d](https://k3d.io/) / k8s)  
 
 ## Development
-- check code quality before a commit:
-```bash
-$ make code-check
-```
 
 ### Build
 
 ```bash
-$ go mod vendor
+make build
 ```
 
-### Test
+### Run tests
```

Comment:
```suggestion
### Run Tests
```
Capitalization guidelines recently changed - we now use title case for headlines.